### ISSUE TITLE
Multi-site feature, after first signup the main site is unavailable.

### ISSUE THREAD

**Issue by SJD-DK** [globaleaks/GlobaLeaks#3391]
### What version of GlobaLeaks are you using?

Latest - new install

### What browser(s) are you seeing the problem on?

All

### What operating system(s) are you seeing the problem on?

Linux

### Describe the issue

I’ve recently installed Globaleaks – when I active the multisite feature, the singup form is available.

But after the first signup has been created and credentials sent out, the main Globaleaks admin is unavailable. – the login screen now displays the newly created multisite as the main site.

**Comment by evilaliv3** [comment]
Thank you for reporting this @SJD-DK 

We are not aware of such a bug on this feature. 

I suspect the bug may lay on a very specific configuration.

Would you please try to reproduce the issue and provide the instructions for its reproducibility?

**Comment by SJD-DK** [comment]
Yes sure, here are the following steps that I followed.

1.	Deployed a VM
2.	Deployed Globaleaks, according to the install guide.
[Installation Guide — GLOBALEAKS 4 documentation](https://docs.globaleaks.org/en/main/setup/InstallationGuide.html)

3.	Enable multisite feature
Settings-Advanced- Checked: Enable multisite feature

4.	Added Danish language.

5.	Configured public DNS 
-	app[.]mydomain[.]com
-	[*].app[.]mydomain[.]com 

6.	Configured hostname (Changed it from IP to dns name)
-	Network – HTTPS – Change Hostname to app[.]mydomain[.]com
-	Configured HTTPS certificate.

7.	Configured Multisite root domain
Sites-Options – filled out Root domain used for secondary sites
Checked Allow users to sign up

-------------------------------------------------------------

When I use the signup form, the new site takes over the primary site.
So, when I try to access app[.]mydomain[.]com\admin, the name of the new site created is being displayed and I’m unable to login with the primary site credentials.

**Comment by evilaliv3** [comment]
This is actually the configuration that we use on our demo (try.globaleaks.org) but this issue do not exist.

which subdomain are you choosing for the tenant when you register it?

**Comment by SJD-DK** [comment]
I'm not sure why, but today the site are there and I can still access the main site.
But I can access the new site, because the certificate isn't signing the new sites.?

Yes, I'm using app.mydomain.com as the host and root for the secondary.
Yesterday I used a different subdomain as the root for secondary sites. - is it recommended to use the same as the primary site?

**Comment by evilaliv3** [comment]
Ok if you confirm that the issue is not there anymore i will close the issue and eventually please add here further details if you encurr in this issue again.

As for your questions:
1. At the moment certificates on the secondary tenants can be only activated manually but we will be soon rolling out a patch where these will be instantiated automatically. For the moment you can eventually acquire a wildcard certificate for the root tenant.
2. We have no particular advice here; using the same domain work better for the moment because you can set a wildcard certificate on the root tenant

**Comment by SJD-DK** [comment]
I managed to recreated the issue again.
I changed to mode (Sites-Options-Mode) to default

![image](https://user-images.githubusercontent.com/128384827/226818517-9cb5c73c-9861-415a-9d2c-848ce7b35b58.png)

![image](https://user-images.githubusercontent.com/128384827/226818853-c0549380-c8c8-4190-ab5f-4a7413218b85.png)

The site name created with Default mode, is named tdtd. - it has now taken over the main site.

After I reverted back to a snapshot before, the main login screen is back
